### PR TITLE
[16.0][MIG] l10n_fr_hr_check_ssnid

### PR DESCRIPTION
**Pull request by syera94**
@rvalyi @alexis-via @bealdav Could you please check this PR? thanks in advance, Syera

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]
approved!

test:

**Comment by rvalyi** [comment]
@syera94 to avoid the module to be flagged as "Beta" status (by default), I think you could also add this key in the `__manifest__.py`:
`"development_status": "Mature",`

File: .oca-port.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "14.0": {
+    "16.0-mig-l10n_fr_hr_check_ssnid": {
+      "l10n_fr_hr_check_ssnid": {
+        "blacklist_pull_requests": {
+          "302": true
+        }
+      }
+    }
+  }
+}
```
Comment (ivantodorovich) [inline .oca-port.json]: Using an old version of oca-port, maybe?
AFAIK this file should be in `.oca/` subfolder, with the latest version

**Review by alexis-via — APPROVED** [review]

File: .oca-port.json (hunk 1 above)
Comment (bealdav) [reply .oca-port.json]: Probably @syera94 could you update oca-port ? Thanks

File: l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+#. module: l10n_fr_hr_check_ssnid
+#: model:ir.model.fields,field_description:l10n_fr_hr_check_ssnid.field_hr_employee____last_update
+msgid "Last Modified on"
+msgstr ""
+
+#. module: l10n_fr_hr_check_ssnid
+#: code:addons/l10n_fr_hr_check_ssnid/models/hr_employee.py:0
+#, python-format
+msgid "The French Social Security Number '%s' is invalid. (%s)"
```
Comment (legalsylvain) [inline l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot]: Could you update the fr.po file according to the change '%s' -> '%(ssnid)s' ?

thanks !

**Review by legalsylvain — COMMENTED** [review]
Diff review / no test.
LGTM. Thanks for porting this module. A remark inline (+I think something to fix regarding the place of the oca-port.json file)

File: .oca-port.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "14.0": {
+    "16.0-mig-l10n_fr_hr_check_ssnid": {
+      "l10n_fr_hr_check_ssnid": {
+        "blacklist_pull_requests": {
+          "302": true
```
Comment (legalsylvain) [inline .oca-port.json]: In fact why did you blacklisted this PR ? 
#302 is fixing a lot of things.

(not sure, though. I don't understand very well this new oca convention).

File: l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot (hunk 2 above)
Comment (bealdav) [reply l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot]: Yes she can but don't you think this the weblate job or oca-translate bot ? 
Not sure ?

Comment (legalsylvain) [reply l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot]: When porting a module, updating the Po according to the change you introduce is a good thing. (One single step)
Specially for l10n_fr modules that have only one traduction by design.

File: .oca-port.json (hunk 3 above)
Comment (syera94) [reply .oca-port.json]: You are probably right in the last version i pushed, there is no oca port. to check.

File: l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot (hunk 2 above)
Comment (syera94) [reply l10n_fr_hr_check_ssnid/i18n/l10n_fr_hr_check_ssnid.pot]: done

File: .oca-port.json (hunk 1 above)
Comment (bealdav) [reply .oca-port.json]: no more old version data

File: l10n_fr_hr_check_ssnid/i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+#. module: l10n_fr_hr_check_ssnid
+#: model:ir.model.fields,field_description:l10n_fr_hr_check_ssnid.field_hr_employee____last_update
+msgid "Last Modified on"
+msgstr ""
+
+#. module: l10n_fr_hr_check_ssnid
+#: code:addons/l10n_fr_hr_check_ssnid/models/hr_employee.py:0
+#, python-format
+msgid "The French Social Security Number '%s' is invalid. (%s)"
+msgstr "Le numéro de sécurité sociale '%s' est invalide. (%s)"
```
Comment (legalsylvain) [inline l10n_fr_hr_check_ssnid/i18n/fr.po]: Could you also update the fr.po ?

Comment (syera94) [reply l10n_fr_hr_check_ssnid/i18n/fr.po]: done

**Review by bealdav — APPROVED** [review]

**Pull request merged**